optuna/samplers/random.py

```diff
@@ -68,6 +69,6 @@ def sample_independent(self, study, trial, param_name, param_distribution):
             return self.rng.randint(param_distribution.low, param_distribution.high + 1)
         elif isinstance(param_distribution, distributions.CategoricalDistribution):
             choices = param_distribution.choices
-            return self.rng.randint(len(choices))
+            return choices[self.rng.randint(len(choices))]
```

Comment:
I think this line can be simpler with `numpy.random.choice` as follows:
```python
return self.rng.choice(choices)
```

Reply:
Cool! Thank you.

Fixed: https://github.com/pfnet/optuna/pull/441/commits/ca979247c3fc3516ca36b1fa8445bfb5a6266493